mars/oscar/backends/context.py

```diff
@@ -71,13 +71,15 @@ def _process_result_message(message: Union[ResultMessage, ErrorMessage]):
 
     async def _wait(self, future: asyncio.Future, address: str, message: _MessageBase):
         try:
-            await asyncio.wait([future])
+            await asyncio.shield(future)
         except asyncio.CancelledError:
             try:
                 await self.cancel(address, message.message_id)
             except CannotCancelTask:
                 # cancel failed, already finished
                 raise asyncio.CancelledError
+        except:  # noqa: E722  # nosec  # pylint: disable=bare-except
+            pass
```

Comment:
Why ignore the exception？

Reply:
> 

To make the result consistent with `async.wait` which does not raises.